core/server/master/src/main/java/alluxio/master/file/FileSystemMasterClientServiceHandler.java

```diff
@@ -300,6 +266,18 @@ public String toString() {
   }
 
   @Override
+  public GetMountTableTResponse getMountTable() throws AlluxioTException {
+    return RpcUtils.callAndLog(LOG, new RpcCallableThrowsIOException<GetMountTableTResponse>() {
+      @Override
+      public GetMountTableTResponse call() throws AlluxioException, IOException {
+          mFileSystemMaster.getMountTable();
```

Comment:
convert the return value from `mFileSystemMaster.getMountTable()` to the thrift format `GetMountTableTResponse`, leveraging `MountPointInfo.toThrift`